rank=06 L00: Traceback (most recent call last):
rank=06 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=06 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=06 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=06 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=06 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=06 L06:     f(args)
rank=06 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=06 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=06 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=06 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=06 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=06 L12:     code = compile(f.read(), fname, 'exec')
rank=06 L13:   File "get_me_coupling_tols.py", line 79
rank=06 L14:     mixer=MixerDif(0.02, 5, 100),
rank=06 L15:     ^
rank=06 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 6): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=22 L00: Traceback (most recent call last):
rank=22 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=22 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=22 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=22 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=22 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=22 L06:     f(args)
rank=22 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=22 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=22 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=22 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=22 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=22 L12:     code = compile(f.read(), fname, 'exec')
rank=22 L13:   File "get_me_coupling_tols.py", line 79
rank=22 L14:     mixer=MixerDif(0.02, 5, 100),
rank=22 L15:     ^
rank=22 L16: SyntaxError: keyword argument repeated
rank=23 L00: Traceback (most recent call last):
rank=23 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=23 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=23 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=23 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=23 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=23 L06:     f(args)
rank=23 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=23 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=23 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=23 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=23 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=23 L12:     code = compile(f.read(), fname, 'exec')
rank=23 L13:   File "get_me_coupling_tols.py", line 79
rank=23 L14:     mixer=MixerDif(0.02, 5, 100),
rank=23 L15:     ^
rank=23 L16: SyntaxError: keyword argument repeated
rank=04 L00: Traceback (most recent call last):
rank=04 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=04 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=04 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=04 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=04 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=04 L06:     f(args)
rank=04 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=04 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=04 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=04 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=04 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=04 L12:     code = compile(f.read(), fname, 'exec')
rank=04 L13:   File "get_me_coupling_tols.py", line 79
rank=04 L14:     mixer=MixerDif(0.02, 5, 100),
rank=04 L15:     ^
rank=04 L16: SyntaxError: keyword argument repeated
rank=13 L00: Traceback (most recent call last):
rank=13 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=13 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=13 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=13 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=13 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=13 L06:     f(args)
rank=13 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=13 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=13 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=13 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=13 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=13 L12:     code = compile(f.read(), fname, 'exec')
rank=13 L13:   File "get_me_coupling_tols.py", line 79
rank=13 L14:     mixer=MixerDif(0.02, 5, 100),
rank=13 L15:     ^
rank=13 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 13): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=17 L00: Traceback (most recent call last):
rank=17 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=17 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=17 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=17 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=17 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=17 L06:     f(args)
rank=17 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=17 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=17 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=17 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=17 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=17 L12:     code = compile(f.read(), fname, 'exec')
rank=17 L13:   File "get_me_coupling_tols.py", line 79
rank=17 L14:     mixer=MixerDif(0.02, 5, 100),
rank=17 L15:     ^
rank=17 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 17): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=18 L00: Traceback (most recent call last):
rank=18 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=18 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=18 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=18 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=18 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=18 L06:     f(args)
rank=18 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=18 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=18 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=18 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=18 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=18 L12:     code = compile(f.read(), fname, 'exec')
rank=18 L13:   File "get_me_coupling_tols.py", line 79
rank=18 L14:     mixer=MixerDif(0.02, 5, 100),
rank=18 L15:     ^
rank=18 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 18): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 22): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 23): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=02 L00: Traceback (most recent call last):
rank=02 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=02 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=02 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=02 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=02 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=02 L06:     f(args)
rank=02 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=02 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=02 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=02 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=02 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=02 L12:     code = compile(f.read(), fname, 'exec')
rank=02 L13:   File "get_me_coupling_tols.py", line 79
rank=02 L14:     mixer=MixerDif(0.02, 5, 100),
rank=02 L15:     ^
rank=02 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 2): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 4): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=11 L00: Traceback (most recent call last):
rank=11 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=11 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=11 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=11 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=11 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=11 L06:     f(args)
rank=11 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=11 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=11 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=11 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=11 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=11 L12:     code = compile(f.read(), fname, 'exec')
rank=11 L13:   File "get_me_coupling_tols.py", line 79
rank=11 L14:     mixer=MixerDif(0.02, 5, 100),
rank=11 L15:     ^
rank=11 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 11): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=03 L00: Traceback (most recent call last):
rank=03 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=03 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=03 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=03 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=03 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=03 L06:     f(args)
rank=03 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=03 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=03 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=03 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=03 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=03 L12:     code = compile(f.read(), fname, 'exec')
rank=03 L13:   File "get_me_coupling_tols.py", line 79
rank=03 L14:     mixer=MixerDif(0.02, 5, 100),
rank=03 L15:     ^
rank=03 L16: SyntaxError: keyword argument repeated
rank=05 L00: Traceback (most recent call last):
rank=05 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=05 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=05 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=05 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=05 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=05 L06:     f(args)
rank=05 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=05 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=05 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=05 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=05 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=05 L12:     code = compile(f.read(), fname, 'exec')
rank=05 L13:   File "get_me_coupling_tols.py", line 79
rank=05 L14:     mixer=MixerDif(0.02, 5, 100),
rank=05 L15:     ^
rank=05 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 5): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=14 L00: Traceback (most recent call last):
rank=14 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=14 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=14 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=14 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=14 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=14 L06:     f(args)
rank=14 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=14 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=14 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=14 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=14 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=14 L12:     code = compile(f.read(), fname, 'exec')
rank=14 L13:   File "get_me_coupling_tols.py", line 79
rank=14 L14:     mixer=MixerDif(0.02, 5, 100),
rank=14 L15:     ^
rank=14 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 14): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=21 L00: Traceback (most recent call last):
rank=21 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=21 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=21 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=21 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=21 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=21 L06:     f(args)
rank=21 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=21 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=21 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=21 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=21 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=21 L12:     code = compile(f.read(), fname, 'exec')
rank=21 L13:   File "get_me_coupling_tols.py", line 79
rank=21 L14:     mixer=MixerDif(0.02, 5, 100),
rank=21 L15:     ^
rank=21 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 21): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 3): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=09 L00: Traceback (most recent call last):
rank=09 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=09 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=09 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=09 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=09 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=09 L06:     f(args)
rank=09 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=09 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=09 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=09 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=09 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=09 L12:     code = compile(f.read(), fname, 'exec')
rank=09 L13:   File "get_me_coupling_tols.py", line 79
rank=09 L14:     mixer=MixerDif(0.02, 5, 100),
rank=09 L15:     ^
rank=09 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 9): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=20 L00: Traceback (most recent call last):
rank=20 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=20 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=20 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=20 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=20 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=20 L06:     f(args)
rank=20 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=20 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=20 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=20 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=20 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=20 L12:     code = compile(f.read(), fname, 'exec')
rank=20 L13:   File "get_me_coupling_tols.py", line 79
rank=20 L14:     mixer=MixerDif(0.02, 5, 100),
rank=20 L15:     ^
rank=20 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 20): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=07 L00: Traceback (most recent call last):
rank=07 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=07 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=07 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=07 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=07 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=07 L06:     f(args)
rank=07 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=07 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=07 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=07 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=07 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=07 L12:     code = compile(f.read(), fname, 'exec')
rank=07 L13:   File "get_me_coupling_tols.py", line 79
rank=07 L14:     mixer=MixerDif(0.02, 5, 100),
rank=07 L15:     ^
rank=07 L16: SyntaxError: keyword argument repeated
rank=08 L00: Traceback (most recent call last):
rank=08 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=08 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=08 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=08 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=08 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=08 L06:     f(args)
rank=08 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=08 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=08 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=08 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=08 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=08 L12:     code = compile(f.read(), fname, 'exec')
rank=08 L13:   File "get_me_coupling_tols.py", line 79
rank=08 L14:     mixer=MixerDif(0.02, 5, 100),
rank=08 L15:     ^
rank=08 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 8): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=16 L00: Traceback (most recent call last):
rank=16 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=16 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=16 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=16 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=16 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=16 L06:     f(args)
rank=16 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=16 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=16 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=16 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=16 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=16 L12:     code = compile(f.read(), fname, 'exec')
rank=16 L13:   File "get_me_coupling_tols.py", line 79
rank=16 L14:     mixer=MixerDif(0.02, 5, 100),
rank=16 L15:     ^
rank=16 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 16): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=01 L00: Traceback (most recent call last):
rank=01 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=01 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=01 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=01 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=01 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=01 L06:     f(args)
rank=01 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=01 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=01 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=01 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=01 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=01 L12:     code = compile(f.read(), fname, 'exec')
rank=01 L13:   File "get_me_coupling_tols.py", line 79
rank=01 L14:     mixer=MixerDif(0.02, 5, 100),
rank=01 L15:     ^
rank=01 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 7): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=12 L00: Traceback (most recent call last):
rank=12 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=12 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=12 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=12 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=12 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=12 L06:     f(args)
rank=12 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=12 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=12 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=12 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=12 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=12 L12:     code = compile(f.read(), fname, 'exec')
rank=12 L13:   File "get_me_coupling_tols.py", line 79
rank=12 L14:     mixer=MixerDif(0.02, 5, 100),
rank=12 L15:     ^
rank=12 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 12): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=15 L00: Traceback (most recent call last):
rank=15 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=15 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=15 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=15 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=15 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=15 L06:     f(args)
rank=15 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=15 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=15 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=15 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=15 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=15 L12:     code = compile(f.read(), fname, 'exec')
rank=15 L13:   File "get_me_coupling_tols.py", line 79
rank=15 L14:     mixer=MixerDif(0.02, 5, 100),
rank=15 L15:     ^
rank=15 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 15): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=19 L00: Traceback (most recent call last):
rank=19 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=19 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=19 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=19 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=19 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=19 L06:     f(args)
rank=19 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=19 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=19 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=19 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=19 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=19 L12:     code = compile(f.read(), fname, 'exec')
rank=19 L13:   File "get_me_coupling_tols.py", line 79
rank=19 L14:     mixer=MixerDif(0.02, 5, 100),
rank=19 L15:     ^
rank=19 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 19): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=00 L00: Traceback (most recent call last):
rank=00 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=00 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=00 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=00 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=00 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=00 L06:     f(args)
rank=00 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=00 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=00 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=00 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=00 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=00 L12:     code = compile(f.read(), fname, 'exec')
rank=00 L13:   File "get_me_coupling_tols.py", line 79
rank=00 L14:     mixer=MixerDif(0.02, 5, 100),
rank=00 L15:     ^
rank=00 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 0): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 1): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=10 L00: Traceback (most recent call last):
rank=10 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=10 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=10 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=10 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=10 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=10 L06:     f(args)
rank=10 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=10 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=10 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=10 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=10 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=10 L12:     code = compile(f.read(), fname, 'exec')
rank=10 L13:   File "get_me_coupling_tols.py", line 79
rank=10 L14:     mixer=MixerDif(0.02, 5, 100),
rank=10 L15:     ^
rank=10 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 10): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=54 L00: Traceback (most recent call last):
rank=54 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=54 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=54 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=54 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=54 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=54 L06:     f(args)
rank=54 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=54 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=54 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=54 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=54 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=54 L12:     code = compile(f.read(), fname, 'exec')
rank=54 L13:   File "get_me_coupling_tols.py", line 79
rank=54 L14:     mixer=MixerDif(0.02, 5, 100),
rank=54 L15:     ^
rank=54 L16: SyntaxError: keyword argument repeated
rank=71 L00: Traceback (most recent call last):
rank=71 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=71 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=71 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=71 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=71 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=71 L06:     f(args)
rank=71 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=71 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=71 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=71 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=71 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=71 L12:     code = compile(f.read(), fname, 'exec')
rank=71 L13:   File "get_me_coupling_tols.py", line 79
rank=71 L14:     mixer=MixerDif(0.02, 5, 100),
rank=71 L15:     ^
rank=71 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 71): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=49 L00: Traceback (most recent call last):
rank=49 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=49 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=49 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=49 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=49 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=49 L06:     f(args)
rank=49 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=49 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=49 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=49 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=49 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=49 L12:     code = compile(f.read(), fname, 'exec')
rank=49 L13:   File "get_me_coupling_tols.py", line 79
rank=49 L14:     mixer=MixerDif(0.02, 5, 100),
rank=49 L15:     ^
rank=49 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 49): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=55 L00: Traceback (most recent call last):
rank=55 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=55 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=55 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=55 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=55 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=55 L06:     f(args)
rank=55 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=55 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=55 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=55 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=55 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=55 L12:     code = compile(f.read(), fname, 'exec')
rank=55 L13:   File "get_me_coupling_tols.py", line 79
rank=55 L14:     mixer=MixerDif(0.02, 5, 100),
rank=55 L15:     ^
rank=55 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 55): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=56 L00: Traceback (most recent call last):
rank=56 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=56 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=56 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=56 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=56 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=56 L06:     f(args)
rank=56 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=56 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=56 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=56 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=56 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=56 L12:     code = compile(f.read(), fname, 'exec')
rank=56 L13:   File "get_me_coupling_tols.py", line 79
rank=56 L14:     mixer=MixerDif(0.02, 5, 100),
rank=56 L15:     ^
rank=56 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 56): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=62 L00: Traceback (most recent call last):
rank=62 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=62 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=62 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=62 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=62 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=62 L06:     f(args)
rank=62 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=62 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=62 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=62 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=62 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=62 L12:     code = compile(f.read(), fname, 'exec')
rank=62 L13:   File "get_me_coupling_tols.py", line 79
rank=62 L14:     mixer=MixerDif(0.02, 5, 100),
rank=62 L15:     ^
rank=62 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 62): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=66 L00: Traceback (most recent call last):
rank=66 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=66 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=66 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=66 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=66 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=66 L06:     f(args)
rank=66 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=66 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=66 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=66 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=66 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=66 L12:     code = compile(f.read(), fname, 'exec')
rank=66 L13:   File "get_me_coupling_tols.py", line 79
rank=66 L14:     mixer=MixerDif(0.02, 5, 100),
rank=66 L15:     ^
rank=66 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 66): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=67 L00: Traceback (most recent call last):
rank=67 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=67 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=67 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=67 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=67 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=67 L06:     f(args)
rank=67 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=67 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=67 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=67 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=67 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=67 L12:     code = compile(f.read(), fname, 'exec')
rank=67 L13:   File "get_me_coupling_tols.py", line 79
rank=67 L14:     mixer=MixerDif(0.02, 5, 100),
rank=67 L15:     ^
rank=67 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 67): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=68 L00: Traceback (most recent call last):
rank=68 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=68 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=68 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=68 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=68 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=68 L06:     f(args)
rank=68 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=68 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=68 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=68 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=68 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=68 L12:     code = compile(f.read(), fname, 'exec')
rank=68 L13:   File "get_me_coupling_tols.py", line 79
rank=68 L14:     mixer=MixerDif(0.02, 5, 100),
rank=68 L15:     ^
rank=68 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 68): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=69 L00: Traceback (most recent call last):
rank=69 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=69 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=69 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=69 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=69 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=69 L06:     f(args)
rank=69 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=69 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=69 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=69 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=69 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=69 L12:     code = compile(f.read(), fname, 'exec')
rank=69 L13:   File "get_me_coupling_tols.py", line 79
rank=69 L14:     mixer=MixerDif(0.02, 5, 100),
rank=69 L15:     ^
rank=69 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 69): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=70 L00: Traceback (most recent call last):
rank=70 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=70 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=70 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=70 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=70 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=70 L06:     f(args)
rank=70 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=70 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=70 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=70 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=70 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=70 L12:     code = compile(f.read(), fname, 'exec')
rank=70 L13:   File "get_me_coupling_tols.py", line 79
rank=70 L14:     mixer=MixerDif(0.02, 5, 100),
rank=70 L15:     ^
rank=70 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 70): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=48 L00: Traceback (most recent call last):
rank=48 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=48 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=48 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=48 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=48 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=48 L06:     f(args)
rank=48 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=48 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=48 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=48 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=48 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=48 L12:     code = compile(f.read(), fname, 'exec')
rank=48 L13:   File "get_me_coupling_tols.py", line 79
rank=48 L14:     mixer=MixerDif(0.02, 5, 100),
rank=48 L15:     ^
rank=48 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 48): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=51 L00: Traceback (most recent call last):
rank=51 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=51 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=51 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=51 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=51 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=51 L06:     f(args)
rank=51 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=51 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=51 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=51 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=51 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=51 L12:     code = compile(f.read(), fname, 'exec')
rank=51 L13:   File "get_me_coupling_tols.py", line 79
rank=51 L14:     mixer=MixerDif(0.02, 5, 100),
rank=51 L15:     ^
rank=51 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 51): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=52 L00: Traceback (most recent call last):
rank=52 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=52 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=52 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=52 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=52 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=52 L06:     f(args)
rank=52 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=52 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=52 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=52 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=52 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=52 L12:     code = compile(f.read(), fname, 'exec')
rank=52 L13:   File "get_me_coupling_tols.py", line 79
rank=52 L14:     mixer=MixerDif(0.02, 5, 100),
rank=52 L15:     ^
rank=52 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 52): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=58 L00: Traceback (most recent call last):
rank=58 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=58 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=58 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=58 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=58 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=58 L06:     f(args)
rank=58 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=58 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=58 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=58 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=58 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=58 L12:     code = compile(f.read(), fname, 'exec')
rank=58 L13:   File "get_me_coupling_tols.py", line 79
rank=58 L14:     mixer=MixerDif(0.02, 5, 100),
rank=58 L15:     ^
rank=58 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 58): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=64 L00: Traceback (most recent call last):
rank=64 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=64 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=64 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=64 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=64 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=64 L06:     f(args)
rank=64 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=64 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=64 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=64 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=64 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=64 L12:     code = compile(f.read(), fname, 'exec')
rank=64 L13:   File "get_me_coupling_tols.py", line 79
rank=64 L14:     mixer=MixerDif(0.02, 5, 100),
rank=64 L15:     ^
rank=64 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 64): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=59 L00: Traceback (most recent call last):
rank=59 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=59 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=59 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=59 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=59 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=59 L06:     f(args)
rank=59 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=59 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=59 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=59 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=59 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=59 L12:     code = compile(f.read(), fname, 'exec')
rank=59 L13:   File "get_me_coupling_tols.py", line 79
rank=59 L14:     mixer=MixerDif(0.02, 5, 100),
rank=59 L15:     ^
rank=59 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 59): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=50 L00: Traceback (most recent call last):
rank=50 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=50 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=50 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=50 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=50 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=50 L06:     f(args)
rank=50 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=50 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=50 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=50 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=50 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=50 L12:     code = compile(f.read(), fname, 'exec')
rank=50 L13:   File "get_me_coupling_tols.py", line 79
rank=50 L14:     mixer=MixerDif(0.02, 5, 100),
rank=50 L15:     ^
rank=50 L16: SyntaxError: keyword argument repeated
rank=53 L00: Traceback (most recent call last):
rank=53 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=53 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=53 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=53 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=53 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=53 L06:     f(args)
rank=53 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=53 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=53 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=53 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=53 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=53 L12:     code = compile(f.read(), fname, 'exec')
rank=53 L13:   File "get_me_coupling_tols.py", line 79
rank=53 L14:     mixer=MixerDif(0.02, 5, 100),
rank=53 L15:     ^
rank=53 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 53): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=57 L00: Traceback (most recent call last):
rank=57 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=57 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=57 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=57 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=57 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=57 L06:     f(args)
rank=57 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=57 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=57 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=57 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=57 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=57 L12:     code = compile(f.read(), fname, 'exec')
rank=57 L13:   File "get_me_coupling_tols.py", line 79
rank=57 L14:     mixer=MixerDif(0.02, 5, 100),
rank=57 L15:     ^
rank=57 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 57): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=65 L00: Traceback (most recent call last):
rank=65 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=65 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=65 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=65 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=65 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=65 L06:     f(args)
rank=65 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=65 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=65 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=65 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=65 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=65 L12:     code = compile(f.read(), fname, 'exec')
rank=65 L13:   File "get_me_coupling_tols.py", line 79
rank=65 L14:     mixer=MixerDif(0.02, 5, 100),
rank=65 L15:     ^
rank=65 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 50): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=63 L00: Traceback (most recent call last):
rank=63 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=63 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=63 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=63 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=63 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=63 L06:     f(args)
rank=63 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=63 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=63 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=63 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=63 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=63 L12:     code = compile(f.read(), fname, 'exec')
rank=63 L13:   File "get_me_coupling_tols.py", line 79
rank=63 L14:     mixer=MixerDif(0.02, 5, 100),
rank=63 L15:     ^
rank=63 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 65): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=60 L00: Traceback (most recent call last):
rank=60 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=60 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=60 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=60 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=60 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=60 L06:     f(args)
rank=60 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=60 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=60 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=60 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=60 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=60 L12:     code = compile(f.read(), fname, 'exec')
rank=60 L13:   File "get_me_coupling_tols.py", line 79
rank=60 L14:     mixer=MixerDif(0.02, 5, 100),
rank=60 L15:     ^
rank=60 L16: SyntaxError: keyword argument repeated
rank=61 L00: Traceback (most recent call last):
rank=61 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=61 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=61 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=61 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=61 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=61 L06:     f(args)
rank=61 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=61 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=61 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=61 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=61 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=61 L12:     code = compile(f.read(), fname, 'exec')
rank=61 L13:   File "get_me_coupling_tols.py", line 79
rank=61 L14:     mixer=MixerDif(0.02, 5, 100),
rank=61 L15:     ^
rank=61 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 61): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 63): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 60): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 54): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=27 L00: Traceback (most recent call last):
rank=27 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=27 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=27 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=27 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=27 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=27 L06:     f(args)
rank=27 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=27 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=27 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=27 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=27 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=27 L12:     code = compile(f.read(), fname, 'exec')
rank=27 L13:   File "get_me_coupling_tols.py", line 79
rank=27 L14:     mixer=MixerDif(0.02, 5, 100),
rank=27 L15:     ^
rank=27 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 27): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=33 L00: Traceback (most recent call last):
rank=33 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=33 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=33 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=33 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=33 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=33 L06:     f(args)
rank=33 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=33 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=33 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=33 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=33 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=33 L12:     code = compile(f.read(), fname, 'exec')
rank=33 L13:   File "get_me_coupling_tols.py", line 79
rank=33 L14:     mixer=MixerDif(0.02, 5, 100),
rank=33 L15:     ^
rank=33 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 33): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=42 L00: Traceback (most recent call last):
rank=42 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=42 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=42 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=42 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=42 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=42 L06:     f(args)
rank=42 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=42 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=42 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=42 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=42 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=42 L12:     code = compile(f.read(), fname, 'exec')
rank=42 L13:   File "get_me_coupling_tols.py", line 79
rank=42 L14:     mixer=MixerDif(0.02, 5, 100),
rank=42 L15:     ^
rank=42 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 42): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=29 L00: Traceback (most recent call last):
rank=29 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=29 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=29 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=29 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=29 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=29 L06:     f(args)
rank=29 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=29 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=29 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=29 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=29 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=29 L12:     code = compile(f.read(), fname, 'exec')
rank=29 L13:   File "get_me_coupling_tols.py", line 79
rank=29 L14:     mixer=MixerDif(0.02, 5, 100),
rank=29 L15:     ^
rank=29 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 29): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=30 L00: Traceback (most recent call last):
rank=30 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=30 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=30 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=30 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=30 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=30 L06:     f(args)
rank=30 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=30 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=30 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=30 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=30 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=30 L12:     code = compile(f.read(), fname, 'exec')
rank=30 L13:   File "get_me_coupling_tols.py", line 79
rank=30 L14:     mixer=MixerDif(0.02, 5, 100),
rank=30 L15:     ^
rank=30 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 30): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=25 L00: Traceback (most recent call last):
rank=25 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=25 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=25 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=25 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=25 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=25 L06:     f(args)
rank=25 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=25 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=25 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=25 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=25 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=25 L12:     code = compile(f.read(), fname, 'exec')
rank=25 L13:   File "get_me_coupling_tols.py", line 79
rank=25 L14:     mixer=MixerDif(0.02, 5, 100),
rank=25 L15:     ^
rank=25 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 25): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=26 L00: Traceback (most recent call last):
rank=26 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=26 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=26 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=26 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=26 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=26 L06:     f(args)
rank=26 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=26 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=26 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=26 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=26 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=26 L12:     code = compile(f.read(), fname, 'exec')
rank=26 L13:   File "get_me_coupling_tols.py", line 79
rank=26 L14:     mixer=MixerDif(0.02, 5, 100),
rank=26 L15:     ^
rank=26 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 26): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=37 L00: Traceback (most recent call last):
rank=37 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=37 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=37 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=37 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=37 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=37 L06:     f(args)
rank=37 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=37 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=37 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=37 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=37 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=37 L12:     code = compile(f.read(), fname, 'exec')
rank=37 L13:   File "get_me_coupling_tols.py", line 79
rank=37 L14:     mixer=MixerDif(0.02, 5, 100),
rank=37 L15:     ^
rank=37 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 37): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=38 L00: Traceback (most recent call last):
rank=38 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=38 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=38 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=38 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=38 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=38 L06:     f(args)
rank=38 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=38 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=38 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=38 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=38 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=38 L12:     code = compile(f.read(), fname, 'exec')
rank=38 L13:   File "get_me_coupling_tols.py", line 79
rank=38 L14:     mixer=MixerDif(0.02, 5, 100),
rank=38 L15:     ^
rank=38 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 38): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=41 L00: Traceback (most recent call last):
rank=41 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=41 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=41 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=41 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=41 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=41 L06:     f(args)
rank=41 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=41 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=41 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=41 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=41 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=41 L12:     code = compile(f.read(), fname, 'exec')
rank=41 L13:   File "get_me_coupling_tols.py", line 79
rank=41 L14:     mixer=MixerDif(0.02, 5, 100),
rank=41 L15:     ^
rank=41 L16: SyntaxError: keyword argument repeated
rank=31 L00: Traceback (most recent call last):
rank=31 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=31 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=31 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=31 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=31 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=31 L06:     f(args)
rank=31 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=31 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=31 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=31 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=31 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=31 L12:     code = compile(f.read(), fname, 'exec')
rank=31 L13:   File "get_me_coupling_tols.py", line 79
rank=31 L14:     mixer=MixerDif(0.02, 5, 100),
rank=31 L15:     ^
rank=31 L16: SyntaxError: keyword argument repeated
rank=32 L00: Traceback (most recent call last):
rank=32 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=32 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=32 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=32 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=32 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=32 L06:     f(args)
rank=32 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=32 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=32 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=32 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=32 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=32 L12:     code = compile(f.read(), fname, 'exec')
rank=32 L13:   File "get_me_coupling_tols.py", line 79
rank=32 L14:     mixer=MixerDif(0.02, 5, 100),
rank=32 L15:     ^
rank=32 L16: SyntaxError: keyword argument repeated
rank=36 L00: Traceback (most recent call last):
rank=36 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=36 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=36 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=36 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=36 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=36 L06:     f(args)
rank=36 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=36 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=36 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=36 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=36 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=36 L12:     code = compile(f.read(), fname, 'exec')
rank=36 L13:   File "get_me_coupling_tols.py", line 79
rank=36 L14:     mixer=MixerDif(0.02, 5, 100),
rank=36 L15:     ^
rank=36 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 36): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=39 L00: Traceback (most recent call last):
rank=39 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=39 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=39 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=39 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=39 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=39 L06:     f(args)
rank=39 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=39 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=39 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=39 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=39 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=39 L12:     code = compile(f.read(), fname, 'exec')
rank=39 L13:   File "get_me_coupling_tols.py", line 79
rank=39 L14:     mixer=MixerDif(0.02, 5, 100),
rank=39 L15:     ^
rank=39 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 39): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=40 L00: Traceback (most recent call last):
rank=40 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=40 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=40 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=40 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=40 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=40 L06:     f(args)
rank=40 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=40 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=40 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=40 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=40 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=40 L12:     code = compile(f.read(), fname, 'exec')
rank=40 L13:   File "get_me_coupling_tols.py", line 79
rank=40 L14:     mixer=MixerDif(0.02, 5, 100),
rank=40 L15:     ^
rank=40 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 40): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 41): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=46 L00: Traceback (most recent call last):
rank=46 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=46 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=46 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=46 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=46 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=46 L06:     f(args)
rank=46 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=46 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=46 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=46 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=46 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=46 L12:     code = compile(f.read(), fname, 'exec')
rank=46 L13:   File "get_me_coupling_tols.py", line 79
rank=46 L14:     mixer=MixerDif(0.02, 5, 100),
rank=46 L15:     ^
rank=46 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 46): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=47 L00: Traceback (most recent call last):
rank=47 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=47 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=47 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=47 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=47 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=47 L06:     f(args)
rank=47 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=47 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=47 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=47 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=47 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=47 L12:     code = compile(f.read(), fname, 'exec')
rank=47 L13:   File "get_me_coupling_tols.py", line 79
rank=47 L14:     mixer=MixerDif(0.02, 5, 100),
rank=47 L15:     ^
rank=47 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 47): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=24 L00: Traceback (most recent call last):
rank=24 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=24 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=24 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=24 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=24 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=24 L06:     f(args)
rank=24 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=24 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=24 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=24 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=24 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=24 L12:     code = compile(f.read(), fname, 'exec')
rank=24 L13:   File "get_me_coupling_tols.py", line 79
rank=24 L14:     mixer=MixerDif(0.02, 5, 100),
rank=24 L15:     ^
rank=24 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 24): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 31): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 32): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=34 L00: Traceback (most recent call last):
rank=34 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=34 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=34 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=34 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=34 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=34 L06:     f(args)
rank=34 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=34 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=34 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=34 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=34 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=34 L12:     code = compile(f.read(), fname, 'exec')
rank=34 L13:   File "get_me_coupling_tols.py", line 79
rank=34 L14:     mixer=MixerDif(0.02, 5, 100),
rank=34 L15:     ^
rank=34 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 34): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=35 L00: Traceback (most recent call last):
rank=35 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=35 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=35 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=35 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=35 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=35 L06:     f(args)
rank=35 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=35 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=35 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=35 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=35 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=35 L12:     code = compile(f.read(), fname, 'exec')
rank=35 L13:   File "get_me_coupling_tols.py", line 79
rank=35 L14:     mixer=MixerDif(0.02, 5, 100),
rank=35 L15:     ^
rank=35 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 35): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=43 L00: Traceback (most recent call last):
rank=43 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=43 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=43 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=43 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=43 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=43 L06:     f(args)
rank=43 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=43 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=43 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=43 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=43 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=43 L12:     code = compile(f.read(), fname, 'exec')
rank=43 L13:   File "get_me_coupling_tols.py", line 79
rank=43 L14:     mixer=MixerDif(0.02, 5, 100),
rank=43 L15:     ^
rank=43 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 43): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=45 L00: Traceback (most recent call last):
rank=45 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=45 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=45 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=45 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=45 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=45 L06:     f(args)
rank=45 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=45 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=45 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=45 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=45 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=45 L12:     code = compile(f.read(), fname, 'exec')
rank=45 L13:   File "get_me_coupling_tols.py", line 79
rank=45 L14:     mixer=MixerDif(0.02, 5, 100),
rank=45 L15:     ^
rank=45 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 45): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=28 L00: Traceback (most recent call last):
rank=28 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=28 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=28 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=28 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=28 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=28 L06:     f(args)
rank=28 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=28 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=28 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=28 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=28 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=28 L12:     code = compile(f.read(), fname, 'exec')
rank=28 L13:   File "get_me_coupling_tols.py", line 79
rank=28 L14:     mixer=MixerDif(0.02, 5, 100),
rank=28 L15:     ^
rank=28 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 28): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=44 L00: Traceback (most recent call last):
rank=44 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=44 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=44 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=44 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=44 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=44 L06:     f(args)
rank=44 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=44 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=44 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=44 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=44 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=44 L12:     code = compile(f.read(), fname, 'exec')
rank=44 L13:   File "get_me_coupling_tols.py", line 79
rank=44 L14:     mixer=MixerDif(0.02, 5, 100),
rank=44 L15:     ^
rank=44 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 44): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=74 L00: Traceback (most recent call last):
rank=74 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=74 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=74 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=74 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=74 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=74 L06:     f(args)
rank=74 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=74 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=74 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=74 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=74 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=74 L12:     code = compile(f.read(), fname, 'exec')
rank=74 L13:   File "get_me_coupling_tols.py", line 79
rank=74 L14:     mixer=MixerDif(0.02, 5, 100),
rank=74 L15:     ^
rank=74 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 74): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=78 L00: Traceback (most recent call last):
rank=78 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=78 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=78 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=78 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=78 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=78 L06:     f(args)
rank=78 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=78 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=78 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=78 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=78 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=78 L12:     code = compile(f.read(), fname, 'exec')
rank=78 L13:   File "get_me_coupling_tols.py", line 79
rank=78 L14:     mixer=MixerDif(0.02, 5, 100),
rank=78 L15:     ^
rank=78 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 78): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=81 L00: Traceback (most recent call last):
rank=81 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=81 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=81 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=81 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=81 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=81 L06:     f(args)
rank=81 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=81 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=81 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=81 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=81 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=81 L12:     code = compile(f.read(), fname, 'exec')
rank=81 L13:   File "get_me_coupling_tols.py", line 79
rank=81 L14:     mixer=MixerDif(0.02, 5, 100),
rank=81 L15:     ^
rank=81 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 81): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=87 L00: Traceback (most recent call last):
rank=87 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=87 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=87 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=87 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=87 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=87 L06:     f(args)
rank=87 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=87 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=87 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=87 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=87 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=87 L12:     code = compile(f.read(), fname, 'exec')
rank=87 L13:   File "get_me_coupling_tols.py", line 79
rank=87 L14:     mixer=MixerDif(0.02, 5, 100),
rank=87 L15:     ^
rank=87 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 87): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=90 L00: Traceback (most recent call last):
rank=90 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=90 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=90 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=90 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=90 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=90 L06:     f(args)
rank=90 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=90 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=90 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=90 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=90 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=90 L12:     code = compile(f.read(), fname, 'exec')
rank=90 L13:   File "get_me_coupling_tols.py", line 79
rank=90 L14:     mixer=MixerDif(0.02, 5, 100),
rank=90 L15:     ^
rank=90 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 90): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=72 L00: Traceback (most recent call last):
rank=72 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=72 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=72 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=72 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=72 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=72 L06:     f(args)
rank=72 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=72 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=72 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=72 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=72 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=72 L12:     code = compile(f.read(), fname, 'exec')
rank=72 L13:   File "get_me_coupling_tols.py", line 79
rank=72 L14:     mixer=MixerDif(0.02, 5, 100),
rank=72 L15:     ^
rank=72 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 72): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=76 L00: Traceback (most recent call last):
rank=76 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=76 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=76 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=76 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=76 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=76 L06:     f(args)
rank=76 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=76 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=76 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=76 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=76 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=76 L12:     code = compile(f.read(), fname, 'exec')
rank=76 L13:   File "get_me_coupling_tols.py", line 79
rank=76 L14:     mixer=MixerDif(0.02, 5, 100),
rank=76 L15:     ^
rank=76 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 76): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=84 L00: Traceback (most recent call last):
rank=84 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=84 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=84 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=84 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=84 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=84 L06:     f(args)
rank=84 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=84 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=84 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=84 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=84 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=84 L12:     code = compile(f.read(), fname, 'exec')
rank=84 L13:   File "get_me_coupling_tols.py", line 79
rank=84 L14:     mixer=MixerDif(0.02, 5, 100),
rank=84 L15:     ^
rank=84 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 84): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=88 L00: Traceback (most recent call last):
rank=88 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=88 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=88 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=88 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=88 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=88 L06:     f(args)
rank=88 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=88 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=88 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=88 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=88 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=88 L12:     code = compile(f.read(), fname, 'exec')
rank=88 L13:   File "get_me_coupling_tols.py", line 79
rank=88 L14:     mixer=MixerDif(0.02, 5, 100),
rank=88 L15:     ^
rank=88 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 88): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=89 L00: Traceback (most recent call last):
rank=89 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=89 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=89 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=89 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=89 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=89 L06:     f(args)
rank=89 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=89 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=89 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=89 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=89 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=89 L12:     code = compile(f.read(), fname, 'exec')
rank=89 L13:   File "get_me_coupling_tols.py", line 79
rank=89 L14:     mixer=MixerDif(0.02, 5, 100),
rank=89 L15:     ^
rank=89 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 89): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=91 L00: Traceback (most recent call last):
rank=91 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=91 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=91 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=91 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=91 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=91 L06:     f(args)
rank=91 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=91 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=91 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=91 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=91 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=91 L12:     code = compile(f.read(), fname, 'exec')
rank=91 L13:   File "get_me_coupling_tols.py", line 79
rank=91 L14:     mixer=MixerDif(0.02, 5, 100),
rank=91 L15:     ^
rank=91 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 91): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=94 L00: Traceback (most recent call last):
rank=94 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=94 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=94 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=94 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=94 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=94 L06:     f(args)
rank=94 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=94 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=94 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=94 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=94 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=94 L12:     code = compile(f.read(), fname, 'exec')
rank=94 L13:   File "get_me_coupling_tols.py", line 79
rank=94 L14:     mixer=MixerDif(0.02, 5, 100),
rank=94 L15:     ^
rank=94 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 94): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=95 L00: Traceback (most recent call last):
rank=95 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=95 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=95 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=95 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=95 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=95 L06:     f(args)
rank=95 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=95 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=95 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=95 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=95 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=95 L12:     code = compile(f.read(), fname, 'exec')
rank=95 L13:   File "get_me_coupling_tols.py", line 79
rank=95 L14:     mixer=MixerDif(0.02, 5, 100),
rank=95 L15:     ^
rank=95 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 95): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=79 L00: Traceback (most recent call last):
rank=79 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=79 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=79 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=79 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=79 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=79 L06:     f(args)
rank=79 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=79 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=79 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=79 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=79 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=79 L12:     code = compile(f.read(), fname, 'exec')
rank=79 L13:   File "get_me_coupling_tols.py", line 79
rank=79 L14:     mixer=MixerDif(0.02, 5, 100),
rank=79 L15:     ^
rank=79 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 79): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=80 L00: Traceback (most recent call last):
rank=80 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=80 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=80 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=80 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=80 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=80 L06:     f(args)
rank=80 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=80 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=80 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=80 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=80 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=80 L12:     code = compile(f.read(), fname, 'exec')
rank=80 L13:   File "get_me_coupling_tols.py", line 79
rank=80 L14:     mixer=MixerDif(0.02, 5, 100),
rank=80 L15:     ^
rank=80 L16: SyntaxError: keyword argument repeated
rank=82 L00: Traceback (most recent call last):
rank=82 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=82 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=82 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=82 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=82 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=82 L06:     f(args)
rank=82 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=82 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=82 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=82 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=82 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=82 L12:     code = compile(f.read(), fname, 'exec')
rank=82 L13:   File "get_me_coupling_tols.py", line 79
rank=82 L14:     mixer=MixerDif(0.02, 5, 100),
rank=82 L15:     ^
rank=82 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 82): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=83 L00: Traceback (most recent call last):
rank=83 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=83 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=83 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=83 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=83 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=83 L06:     f(args)
rank=83 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=83 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=83 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=83 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=83 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=83 L12:     code = compile(f.read(), fname, 'exec')
rank=83 L13:   File "get_me_coupling_tols.py", line 79
rank=83 L14:     mixer=MixerDif(0.02, 5, 100),
rank=83 L15:     ^
rank=83 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 83): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=85 L00: Traceback (most recent call last):
rank=85 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=85 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=85 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=85 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=85 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=85 L06:     f(args)
rank=85 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=85 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=85 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=85 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=85 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=85 L12:     code = compile(f.read(), fname, 'exec')
rank=85 L13:   File "get_me_coupling_tols.py", line 79
rank=85 L14:     mixer=MixerDif(0.02, 5, 100),
rank=85 L15:     ^
rank=85 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 85): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=92 L00: Traceback (most recent call last):
rank=92 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=92 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=92 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=92 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=92 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=92 L06:     f(args)
rank=92 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=92 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=92 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=92 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=92 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=92 L12:     code = compile(f.read(), fname, 'exec')
rank=92 L13:   File "get_me_coupling_tols.py", line 79
rank=92 L14:     mixer=MixerDif(0.02, 5, 100),
rank=92 L15:     ^
rank=92 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 92): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=75 L00: Traceback (most recent call last):
rank=75 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=75 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=75 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=75 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=75 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=75 L06:     f(args)
rank=75 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=75 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=75 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=75 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=75 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=75 L12:     code = compile(f.read(), fname, 'exec')
rank=75 L13:   File "get_me_coupling_tols.py", line 79
rank=75 L14:     mixer=MixerDif(0.02, 5, 100),
rank=75 L15:     ^
rank=75 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 75): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=77 L00: Traceback (most recent call last):
rank=77 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=77 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=77 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=77 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=77 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=77 L06:     f(args)
rank=77 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=77 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=77 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=77 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=77 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=77 L12:     code = compile(f.read(), fname, 'exec')
rank=77 L13:   File "get_me_coupling_tols.py", line 79
rank=77 L14:     mixer=MixerDif(0.02, 5, 100),
rank=77 L15:     ^
rank=77 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 80): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=86 L00: Traceback (most recent call last):
rank=86 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=86 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=86 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=86 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=86 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=86 L06:     f(args)
rank=86 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=86 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=86 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=86 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=86 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=86 L12:     code = compile(f.read(), fname, 'exec')
rank=86 L13:   File "get_me_coupling_tols.py", line 79
rank=86 L14:     mixer=MixerDif(0.02, 5, 100),
rank=86 L15:     ^
rank=86 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 86): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=93 L00: Traceback (most recent call last):
rank=93 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=93 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=93 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=93 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=93 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=93 L06:     f(args)
rank=93 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=93 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=93 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=93 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=93 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=93 L12:     code = compile(f.read(), fname, 'exec')
rank=93 L13:   File "get_me_coupling_tols.py", line 79
rank=93 L14:     mixer=MixerDif(0.02, 5, 100),
rank=93 L15:     ^
rank=93 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 93): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 77): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
rank=73 L00: Traceback (most recent call last):
rank=73 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=73 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=73 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=73 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=73 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=73 L06:     f(args)
rank=73 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=73 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=73 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 264, in run_path
rank=73 L10:     code, fname = _get_code_from_file(run_name, path_name)
rank=73 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 239, in _get_code_from_file
rank=73 L12:     code = compile(f.read(), fname, 'exec')
rank=73 L13:   File "get_me_coupling_tols.py", line 79
rank=73 L14:     mixer=MixerDif(0.02, 5, 100),
rank=73 L15:     ^
rank=73 L16: SyntaxError: keyword argument repeated
GPAW CLEANUP (node 73): <class 'SyntaxError'> occurred.  Calling MPI_Abort!
Abort(42) on node 7 (rank 7 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 7
